client/src/components/APIDataTypes.ts

```diff
@@ -139,6 +139,9 @@ export type SummaryStatsRecord = {
   topbusinessaddr: string | null;
   topcorp: string | null;
   topowners: string[];
+  totalcomplaints: number;
```

Comment:
any chance you want to call this total311complaints, like we did for hpd violations? if there are any other types of complaints it might be harder to refactor later

Reply:
Or will this number continue to be the absolute total even if we added other types of complaints?

Reply:
good point! you're right that I should name this more specifically. I'll use `totalhpdcomplaints` bc that's what we call it internally